src/main/java/org/usfirst/frc/team2473/robot/commands/AlignToHatch.java

```diff
@@ -12,66 +12,42 @@
 
 import java.io.IOException;
 
+import org.usfirst.frc.team2473.framework.Devices;
 import org.usfirst.frc.team2473.framework.UtilitySocket;
 
 /**
  * A class that aligns the robot to the hatch based on the angle provided by CV
  */
 public class AlignToHatch extends Command {
-		
-	double theta;
 	
 	double normalPower = 0.25;
-	double addedPower = 0.15;
-		
-	private UtilitySocket s;
-		
+    double addedPower = 0.15;
+    		
 	public AlignToHatch() {
 		requires(Robot.driveSubsystem);
-		
-		try {
-			System.out.println("Creating utility socket in AlignToHatch");
-			s = new UtilitySocket("10.24.73.19", 5801);
-			System.out.println("Utility socket created!");
-		} catch (IOException e) {
-			System.out.println("Error in Constructor try-catch");
-			e.printStackTrace();
-		}
-	}
-	
-	private void updateTheta() {
-		String thetaStr = s.getLine();
-		if (thetaStr == null) {
-			System.out.println("No new value of theta");
-		} else {
-			theta = Double.parseDouble(thetaStr);
-		}
-		
 	}
 	
 	@Override
 	protected void initialize() {
-		updateTheta();
+		Robot.driveSubsystem.drive(normalPower, normalPower, normalPower + addedPower, normalPower + addedPower);
```

Comment:
The robot should not be turning before we check for the angle. This line isn't necessary, since the robot will start moving after an angle check in the first run of `execute()`